articles/virtual-machines/windows/expand-os-disk.md

```diff
@@ -69,6 +69,10 @@ Open your Powershell ISE or Powershell window in administrative mode and follow
    > The new size should be greater than the existing disk size. The maximum allowed is 2048 GB for OS disks. (It is possible to expand the VHD blob beyond that size, but the OS will only be able to work with the first 2048 GB of space.)
    > 
    > 
+   For managed disk created as copy of disk from different subscription, use below with desired value: 

```

Comment:
@VijayRod could you please provide more context around as to why the specific step is needed?

Reply:
@VijayRod due to lack of context and further details (a quick search around didn't yield any information) about the step you recommended in your PR #16401, I need to close for now, but feel free to reopen with more context and I'll be glad to take another look at it. Thank you for your time reviewing this doc!

#please-close

Reply:
I'm copy you in email with Disk team (Vikram).